assets/app/styles/_core.less

```diff
@@ -104,6 +104,14 @@ html, body {
       margin-bottom: 0;
     }
   }
+  .active-filters {
+    padding-top: 0;
+  }
+  .create-button {
+    white-space: nowrap;
+    text-align: right;
+    margin: 30px 0;
```

Comment:
@liggitt That's not new actually, although it was done using a style attr in the HTML before. It's to align the button vertically with the project dropdown and filter fields. The other fields have a label above the input control.